include/flamegpu/util/cleanup.h

```diff
@@ -1,8 +1,8 @@
 #ifndef INCLUDE_FLAMEGPU_UTIL_CLEANUP_H_
 #define INCLUDE_FLAMEGPU_UTIL_CLEANUP_H_
 
-/**
```

Comment:
Same here, also inconsistent with other changes in this PR.

Reply:
its a file level multline comment, not a docstring about a particular namespace nor class (there isn't a cleanup specific namespace or class, just method(s)